xstate/machine.py

```diff
@@ -22,7 +22,12 @@ def transition(self, state: State, event: str):
 
         value = get_state_value(self.root, configuration=configuration)
 
-        return State(value, context={}, actions=actions)
+        return State(configuration=configuration, context={}, actions=actions)
+
+    def state_from(self, state_value) -> State:
```

Comment:
This is necessary because we're no longer building the `State` from a value, but rather from which of the state nodes are active (the `configuration`). But it's still useful to get a state from a value, e.g., `green_state = light_machine.state_from("green")`.